### ISSUE TITLE
Supply NuGet server URL to deployment script

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1457]
Automatic deploy [for 4.9.0 failed](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/build/667#L488):

```
tools\deploy.cmd
Unhandled Exception: System.Exception: NuGet server URL is not set; please set it in the NUGET_SERVER_URL environment variable.
   at FakeItEasy.Deploy.Program.<Main>d__0.MoveNext() in C:\projects\fakeiteasy\tools\FakeItEasy.Deploy\Program.cs:line 30
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at FakeItEasy.Deploy.Program.<Main>(String[] args)
Command exited with code -532462766
```

**Comment by thomaslevesque** [comment]
Ah. The URl was hard-coded in the PowerShell script. I must have introduced the variable when I migrated the script to csproj and forgot to set it in appveyor.yml 😳

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 4.9.1](https://github.com/FakeItEasy/FakeItEasy/releases/4.9.1).
